frontend/src/components/Label/Labels.tsx

```diff
@@ -8,6 +8,7 @@ const SHOW_MORE_TRESHOLD = 2;
 
 interface LabelsProps {
   expanded?: boolean;
+  header?: string;
```

Comment:
Replace `header` by `type` like in `WizardLabels` component

Reply:
ok, but keeping it as optional with default value, as this component is mostly used in other places for labels